modules/store/spec/selector.spec.ts

```diff
@@ -143,17 +143,6 @@ describe('Selectors', () => {
       });
     });
 
-    it('should not short circuit to the projector fn if there are no selectors and props', () => {
```

Comment:
This test was to verify the fix for #1501.
Since we don't short circuit anymore, this test became obsolete.